app/controllers/test/components/mentoring/mentoring_panel_list_controller.rb

```diff
@@ -1,4 +1,4 @@
-class Test::Components::Mentoring::DiscussionPostPanelController < Test::BaseController
+class Test::Components::Mentoring::MentoringPanelListController < Test::BaseController
```

Comment:
Do we need the double "Mentoring::Mentoring"?

Reply:
I've will fix this in my upcoming PR where I put everything into one component.